Multisig/UI/Transactions/TransactionDetails/TransactionDetailsView.swift

```diff
@@ -92,27 +143,39 @@ struct TransactionDetailsView: Loadable {
         }
     }
 
-    func browseTransaction() -> some View {
+    private var viewTxOnEtherscan: some View {
```

Comment:
Could we extract this view into new component to UI library ?

Reply:
do we use it in other places?